### PR TITLE
NF-512 - allow zero value for "should have paid"

### PR DESCRIPTION
**Pull request by SteveSugden**

### CODE REVIEW

File: test/uk/gov/hmrc/nationalimportdutyadjustmentcentrefrontend/forms/create/DutyPaidFormProviderSpec.scala [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    behave like mandatoryField(form, fieldName, requiredError = FormError(fieldName, requiredKey))
+
+    behave like fieldThatPreventsUnsafeInput(
+      form,
+      fieldName,
+      nonNumerics,
+      FormError(fieldName, invalidKey, Seq(Validation.dutyPattern))
+    )
+
+    "error when input is negative" in {
```
Comment (njmoorewitchcat) [inline test/uk/gov/hmrc/nationalimportdutyadjustmentcentrefrontend/forms/create/DutyPaidFormProviderSpec.scala:70]: OBSERVATION: As this is a new test around a boundary condition, I would have liked to see tests that cover that.  So a zero test that passes, a -0.01 test that fails.  Maybe also a -0.000000001 that fails and a 0.00 that passes (as these are string fields)

**Review by njmoorewitchcat — APPROVED** [review]

**Review by njmoorewitchcat — APPROVED** [review]
